### ISSUE TITLE
support for int?

### ISSUE THREAD

**Issue by real-bombinho** [hsaturn/TinyConsole#3]
Hello and thank you for your work,

would it be possible to support int as well?

suggestion: add TinyString& operator +=(int);

This makes the unit usable on RP2040 (earlephilhower's Arduino board adaption)  as well.

**Comment by hsaturn** [comment]
TinyString should not be used. Unit test are not passing on github (while they are ok on my computer, which is precisely the reason for which they should not be used).
But I take the point. This will be done once unit test are ok, but for the moment I have a lot more important to deal with the library.

Thanks for the suggestion

**Comment by real-bombinho** [comment]
The reason for my usage is the dependency in TinyMqtt. It compiles perfectly fine on my machine(s) too. And for RP2040 it compiles nicely with the suggested change.

No rush, all working here anyway.

**Comment by hsaturn** [comment]
You may think about upgrading to the latest version. Because I know there is a problem with TinyString. Probably a memory fault or something like this.

I've removed the dependency with TinyString in latest versions because of this.

Using TinyString is not safe. Once fixed, TinyString will save memory but yet, try to avoid it.

**Comment by real-bombinho** [comment]
Can I suggest commenting or deleting in TinyConsole.h line 3 '#include "TinyString.h" ' ?

If I understand correctly then this library is not in use and TinyMqtt in the latest version compiles (and works) as expected, once this line is commented.
I have excluded TinyString from my test bed and it throws errors otherwise as it appears to attempt using TinyString from the installed 0.9.6

**Comment by hsaturn** [comment]
Yes, 0.9.6 uses TinyString, but this is not the latest version. In fact TinyMqtt 1.0.0 is released but not handled by IDE's and this is a chance, because 1.0.0 is buggy (not revealed by unit tests :-()

TinyString makes TinyConsole unit tests fails on github, while they are running ok under Linux... Strange behaviour that makes me think about a memory corruption.

TinyString should be replaced with std::string with no error.

```
If I understand correctly then this library is not in use and TinyMqtt in the latest version compiles (and works) as expected, once this line is commented.
```

This lib is used with my roomba project (at least).
